### PR TITLE
Sign Up Button Design Enhancement

### PR DESCRIPTION
**Pull request by iamRabia-N**
This pull request contains the updated code for '[FEATURE] -Sign Up Button Design Enhancement'. 
The issue related to this pull request is as follows:

[FEATURE] -Sign Up Button Design Enhancement #1171

 # Output Screenshot

ip (3)

### CODE REVIEW

**Review by narayan954 — CHANGES_REQUESTED** [review]
Hi @iamRabia-N can you please check this issue?
image

**Comment by iamRabia-N** [comment]
Hi @narayan954 
The bug is resolved in commit with id '2bae451299504e370ff83431dd775be27a560b8e'.

# Output Screenshot

Screenshot (2679)

Screenshot (2678)

**Comment by narayan954** [comment]
> Hi @narayan954 The bug is resolved in commit with id '2bae451299504e370ff83431dd775be27a560b8e'.
> 
> # Output Screenshot
> Screenshot (2679)
> 
> Screenshot (2678)

thanks a lot! :)

File: src/components/Suggestions/index.css [hunk 1]
```diff
@@ -33,6 +33,7 @@
   padding: 0;
   margin: 0;
   scroll-behavior: smooth;
+  margin-top: 7rem;
```
Comment (narayan954) [inline src/components/Suggestions/index.css:36]: you should've checked it for signed in user as well. It's looking bad for logged in user now
image

**Review by narayan954 — CHANGES_REQUESTED** [review]

**Comment by iamRabia-N** [comment]
Then we can settle it in around 4rem so that with and without login it. will look fine.

File: src/components/Suggestions/index.css (hunk 1 above)
Comment (iamRabia-N) [reply src/components/Suggestions/index.css:36]: Then we can settle it in around 4rem so that with and without login, it will look fine.

Comment (narayan954) [reply src/components/Suggestions/index.css:36]: if that looks fine, it's a green light from my side

Comment (narayan954) [reply src/components/Suggestions/index.css:36]: also, do update your branch with the latest changes

**Review by iamRabia-N — COMMENTED** [review]
Done with branch and code updation

**Review by narayan954 — CHANGES_REQUESTED** [review]
Hi @iamRabia-N still the css is breaking layout, as you can see 
image
and 
image

Can't we do something like, create a separate class for this margin-top and add this class only when user is anonymous?

**Comment by iamRabia-N** [comment]
Yeah, I am working on implementing this logic.

**Comment by narayan954** [comment]
> Hi @iamRabia-N still the css is breaking layout, as you can see image and image
> 
> Can't we do something like, create a separate class for this margin-top and add this class only when user is anonymous?

also to give you more info, in suggestion component, we have variable currentUser and it has a method currentUser.isAnonymous, which tells if it's an anonymous user, use this logic to add the separate class to the component

**Comment by iamRabia-N** [comment]
Hi @narayan954 
I have tried the logic (we discussed yesterday) in jsx file . Its' not working.

**Comment by narayan954** [comment]
> Hi @narayan954 I have tried the logic (we discussed yesterday) in jsx file . Its' not working.

Hi @iamRabia-N I checked and was able to solve it using that logic only, Do allow me to edit this pr and I'll push the fixes. 
Also, there're some errors in routing that was causing another issue, I fixed those in those commits as well!

**Comment by narayan954** [comment]
around here @iamRabia-N  you'll get the option to enable `edits by maintainer` option
image

It's a good practise that you enable modifications by maintainer.

**Comment by iamRabia-N** [comment]
@narayan954 You can edit this pull request now.

**Review by narayan954 — APPROVED** [review]
lgtm! but do check the changes I made and learn what you did wrong?

**Pull request merged**